internal/lib/container_server.go

```diff
@@ -151,11 +151,11 @@ func (c *ContainerServer) LoadSandbox(id string) error {
 	}
 	var m rspec.Spec
 	if err := json.Unmarshal(config, &m); err != nil {
-		return err
+		return errors.Wrap(err, "error unmarshalling sandbox spec")
 	}
 	labels := make(map[string]string)
 	if err := json.Unmarshal([]byte(m.Annotations[annotations.Labels]), &labels); err != nil {
-		return err
+		return errors.Wrapf(err, "error unmarshalling %s annotation", annotations.Labels)
```

Comment:
Might be good to add a further descriptor to each of these lines so that they're unique.  It will make searching for the error easier if/when it pops.
Suggest: "%s annotation" -> "%s annotation labels"
for this one, then metadata, portmappings and namespaceoptons for 2, 4, and 5 below.  I the third, annotations at line 182, is fine as is.

Reply:
Hm, not sure what you mean here. It already add the problematic annotation to the log message. It looks like this:

`error unmarshalling io.kubernetes.cri-o.Labels annotation`
`error unmarshalling io.kubernetes.cri-o.Metadata annotation`

 
are you suggesting something like:

`error unmarshalling io.kubernetes.cri-o.Labels annotation labels`
`error unmarshalling io.kubernetes.cri-o.Metadata annotation metadata`

Does that really add any value?

Reply:
My bad, I'd not picked up a description would be there.  I thought it was just going to be a the value like an id or some such.  No change needed and thanks!